participants/harmeet_singh.json

```
@@ -0,0 +1,50 @@
+// vim: ft=jsonc
+// Please provide your info in your own .json file.
+// See https://jscraftcamp.org/registration for more information
+{
+	// your real name (required by location host)
+	"realName": {
+		"givenName": "Harmeet",
+		"familyName": "Singh",
+		// if you prefer to have your family name shown first (optional)
+		"placeFamilyNameFirst": true,
+		// if you do not want to show your family name on the participant list (optional)
+		"hideFamilyNameOnWebsite": false
+	},
+	// please put in the account name of the PR creator, if you sign up somebody else
+	"githubAccountName": "harmeet24",
+	// company name (optional)
+	"company": "Autoscout24",
+	// either both days or at least one day has to be set to true
+	"when": {
+		// June 7th, 2024
+		"friday": true,
+		// June 8th, 2024
+		"saturday": true
+	},
+	// if you are willing to take session notes and publish them to github (required)
+	"iCanTakeNotesDuringSessions": true,
+	// your current interests (JS and in general) (required)
+	"tags": ["Vanilla JS", "Type Script", "Functional Programming", "React", "PureScript", "ELM", "WebAssembly"],
+	// if you only eat vegan food (optional)
+	"vegan": false,
+	// if you only eat vegan or vegetarian food (optional)
+	"vegetarian": true,
+	// what you cannot eat or drink (optional); If you don't want to put it in here, message the organizers.
+	// IMPORTANT: we cannot guarantee that food for every diet will be available,
+	//            if you have gluten free diet, please make backup plans.
+	"allergies": [],
+	// tell us a few words how JavaScript affects you (required)
+	"whatIsMyConnectionToJavascript": "From the beginning of my career, I never get a chance to work on JavaScript and explore JavaScript-based technologies.",
+	// what can you contribute to the bar camp (required)
+	"whatCanIContribute": "I am happy to discuss backend technologies like Java and Scala and happy to explore distributed systems architecture."
+	// if you want a T-Shirt we need your size and variant preference (optional)
+	// the following sizes are available: M (only regular cut)
+	"tShirtSize": "M",
+	// your LinkedIn profile URL
+	"linkedin": "https://www.linkedin.com/in/harmeet-singh-taara-1654165b/",
+	// your X (Twitter) handle - must match regex ^[a-zA-Z_]{1}[a-zA-Z0-9_]{0,14}$ (optional)
+	"X": "@singh_harmeet13",
```

I think the `@` sign needs to be removed here

```suggestion
	"X": "singh_harmeet13",
```